pkg/apis/seedmanagement/v1alpha1/defaults_managedseed_test.go

```diff
@@ -166,6 +181,11 @@ var _ = Describe("Defaults", func() {
 											Namespace: "bar",
 										},
 									},
+									Ingress: &gardencorev1beta1.Ingress{
+										Controller: gardencorev1beta1.IngressController{
+											Kind: "nginx",
+										},
+									},
```

Comment:
Lets verify that an already specified ingress configuration would not be overwritten instead